napari_hub_cli/checklist/metadata.py

```diff
@@ -289,3 +296,13 @@ def display_checklist(analysis_result):
             style="red",
         )
         console.print(f"  Recommended file location - {feature.meta.advise_location}")
+    
+    # Display additional informations
+    for feature in analysis_result.additionals:
+        if feature.meta.section:
+            if feature.meta.section.title != previous_title:
```

Comment:
@SimaoBolota-MetaCell Same as in the code before, if you can compact them in a single `if` using `and` that would be perfect :)